[20.09] Electron updates

prusnak opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change

* https://www.electronjs.org/releases/stable?version=10
* https://www.electronjs.org/releases/stable?version=11
* https://www.electronjs.org/releases/stable?version=12

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

TredwellGit committed:
electron_12: 12.0.5 -> 12.0.6

https://github.com/electron/electron/releases/tag/v12.0.6
(cherry picked from commit 1d0bf51a6e8c9ca21a9a07f271971528813b05f3)

prusnak committed:
electron_10: 10.4.4 -> 10.4.5

(cherry picked from commit d4c3ac5c50234cd44276e2814600e1ff2950535a)

prusnak committed:
electron_11: 11.4.4 -> 11.4.5

(cherry picked from commit 547cf3fbc28dd55d3716698a6501b65b7b75f3a8)

r-rmcgibbo commented:
Result of `nixpkgs-review pr 123109` at eab4b442 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built successfully:</summary>
  <ul>
    <li>electron (electron_11)</li>
    <li>electron_10</li>
    <li>electron_12</li>
    <li>element-desktop</li>
    <li>nix-tour</li>
  </ul>
</details>

TredwellGit committed:
electron_12: 12.0.6 -> 12.0.7

https://github.com/electron/electron/releases/tag/v12.0.7
(cherry picked from commit 537c040cebe877774bce9169001a0978d8db35d7)

TredwellGit committed:
electron_11: 11.4.5 -> 11.4.6

https://github.com/electron/electron/releases/tag/v11.4.6
(cherry picked from commit 88772a7a42a7d9bed9b11662b86fdd89e1b4e95d)

TredwellGit committed:
electron_12: 12.0.7 -> 12.0.9

https://github.com/electron/electron/releases/tag/v12.0.8
https://github.com/electron/electron/releases/tag/v12.0.9
(cherry picked from commit a0426609c8cbf16e8ef1bbae34faa060fec6fe05)

TredwellGit committed:
electron_11: 11.4.6 -> 11.4.7

https://github.com/electron/electron/releases/tag/v11.4.7
(cherry picked from commit 505298f812a263244c2189549933c02d810965a7)

TredwellGit committed:
electron_10: 10.4.5 -> 10.4.7

https://github.com/electron/electron/releases/tag/v10.4.6
https://github.com/electron/electron/releases/tag/v10.4.7
(cherry picked from commit f8fbfa538bcb8a8c187f4d60946cf8f0dc9dc6a7)
